filebrowser_safe/storage.py

```diff
@@ -125,7 +126,7 @@ def rmtree(self, name):
 class GoogleStorageMixin(StorageMixin):
 
     def isfile(self, name):
-        return self.exists(name)
+        return bool(name) and self.exists(name)
```

Comment:
by relying only on `exists()`, the method `isfile()` inappropriately considered the bucket root to be a file.